modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1384,11 +1385,16 @@ private void _processAttachment(
 		FileEntry fileEntry = ObjectMapperUtil.readValue(
 			FileEntry.class, propertyValue);
 
-		if ((fileEntry == null) || (fileEntry.getFileBase64() == null)) {
+		if ((fileEntry == null) ||
+			((fileEntry.getFileBase64() == null) &&
```

Comment:
Hi @devpedrotavares , can we reorder here?

Reply:
Yes, done!